source/layouts/partials/_body_js.html.erb

```diff
@@ -8,3 +8,13 @@
 <% if (locals[:include_addthis] || current_page.data.include_addthis) && ApplicationConfig.const_defined?(:AddThis) && ApplicationConfig::AddThis::ID %>
   <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=<%= ApplicationConfig::AddThis::ID %>"  async="async"></script>
 <% end %>
+
+<% if locals[:enable_intercom] == true && ApplicationConfig.const_defined?(:Intercom) && ApplicationConfig::Intercom::APP_ID %>
```

Comment:
You could omit `== true`.

Aside from that, I would not check if the constant is defined (using `ApplicationConfig.const_defined?(:Intercom)`). If it's not I would like to see an error popping up, otherwise the sync to Intercom only silently doesn't work (or you need to to a test for it).

Reply:
Yeah, over-protection on the `== true`, don't disagree.

On `ApplicationConfig.const_defined?(:Intercom)`, what would I do in https://github.com/sharesight/www.sharesight.com/blob/master/config/environments/development.rb where I do not have a `module Intercom`?  I think that's what this is protecting.

Should I have something like?
```rb
module Intercom
  # NOTE: Unavailable in this environment.
end
```

Reply:
Ah, okay, now I understand what is done here. Yes, your way is correct then. 👍